src/algorithms/tracking/TrackParamTruthInitConfig.h

```diff
@@ -15,6 +15,8 @@ struct TrackParamTruthInitConfig {
     double m_minMomentum      = 100 * Acts::UnitConstants::MeV;
     double m_maxEtaForward    = 4.0;
     double m_maxEtaBackward   = 4.1;
+    double m_momentumSplit    = 0.0;
```

Comment:
Shouldn't we set these to be nonzero by default? Since we know that the kalman filter can be biased by too good of input seed parameters?

Reply:
I believe this file should be modified. And used. That is exactly what I wrote above to @wdconinc

Reply:
I think it is used, my suggestion was just to set the default behavior to use nonzero smearing

Reply:
> the intent was to provide this functionality first to then test what the smearing value should be and then implement that as the default in a future PR

Yes :-)

> I think [this file] is used

Yes.